### PR TITLE
Feature/log activity cards

### PR DESCRIPTION
**Pull request by janitastic**
## Subject:
Activity Cards
## Reason for Pull Request:
Added Activity Cards CSS and JS
## Changes Implemented:
- CSS line, flexbox, and font, etc classes
- JS screen view functions
- Added new buttons to our HTML

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -145,6 +150,40 @@ function updateCountdown() {
...
+  activityCards.innerHTML = '';
+  for (var i = 0; i < pastActivities.length; i++) {
+      activityCards.innerHTML += `
+        <section class="card">
+          <article class="activity-card">
+          <div class="cat-time"><span class="card-text">${pastActivities[i].category}</span><br>
+          ${pastActivities[i].minutes} MIN ${pastActivities[i].seconds} SECONDS<br>
+          <span style="font-size: 12px;">${pastActivities[i].description}</span>
+          </div>
+          <div class="line" id="line">|</div>
```
Comment (LKessell) [inline main.js]:
```suggestion
          <div class="line ${pastActivities[i].category}-line" id="line">|</div>
```

Comment (LKessell) [reply main.js]: This will make our line match the category color!

File: main.js [hunk 2]
```diff
@@ -145,6 +150,40 @@ function updateCountdown() {
...
+          </article>
+        </section>
+     `
+     // if (pastActivities[i].category === 'study') {
+     //   startTimerButton.classList.add('line');
+     // } else if (currentActivity.category === 'meditate') {
+     //   startTimerButton.classList.add('meditate-timer');
+     // } else {
+     //   startTimerButton.classList.add('exercise-timer');
+     // }
```
Comment (LKessell) [inline main.js]: We could also remove these, I found a possible solution for our line!

File: styles.css [hunk 3]
```diff
@@ -410,7 +410,7 @@ main {
 }
 
 .line {
-  font-weight: bold;
```
Comment (Andrey-1992) [inline styles.css]: Good changes of value, to make it more dynamic  !

File: main.js [hunk 4]
```diff
@@ -145,6 +154,50 @@ function updateCountdown() {
...
+}
+
+function clearForm() {
+  // descriptionInput.value = '';
+  // minuteInput.value = '';
+  // secondInput.value = '';
+  // studyIcon.value = '';
+  // meditateIcon.value = '';
+  // exerciseIcon.value = '';
+  window.location.reload();
```
Comment (LKessell) [inline main.js:202]: I think this will work really well after we implement local storage!

File: main.js [hunk 5]
```diff
@@ -159,15 +162,17 @@ function updateCountdown() {
...
   render();
   console.log(currentActivity);
 }
 
 function render() {
-  activityCards.innerHTML = '';
+  cardContent.innerHTML = '';
   for (var i = 0; i < pastActivities.length; i++) {
-      activityCards.innerHTML += `
+      cardContent.innerHTML += `
```
Comment (Andrey-1992) [inline main.js:174]: Good id changes !!

**Review by Andrey-1992 — APPROVED** [review]

File: main.js [hunk 6]
```diff
@@ -145,6 +154,50 @@ function updateCountdown() {
...
+          ${pastActivities[i].minutes} MIN ${pastActivities[i].seconds} SECONDS<br>
+          <span style="font-size: 12px;">${pastActivities[i].description}</span>
+     `
+     if (pastActivities[i].category === 'study') {
+       categoryLine.classList.add('study-line');
+     } else if (pastActivities[i].category === 'meditate') {
+       categoryLine.classList.add('meditate-line');
+     } else {
+       categoryLine.classList.add('exercise-line');
+     }
```
Comment (LKessell) [inline main.js:185]: I have an idea on how we might refactor this in the future!

**Review by LKessell — APPROVED** [review]
Looks great, love the new CSS organization!

**Pull request merged**